<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExchangeEditDlg</class>
 <widget class="QDialog" name="ExchangeEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exchange Edit</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="listExchanges">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cmbName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="keyLabel">
         <property name="text">
          <string>Key</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editKey"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="secretLabel">
         <property name="text">
          <string>Secret</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editSecret">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>316</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnNew">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDelete">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnClose">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>ExchangeEditDlg</receiver>
   <slot>OnClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listExchanges</sender>
   <signal>itemActivated(QListWidgetItem*)</signal>
   <receiver>ExchangeEditDlg</receiver>
   <slot>OnExchangeSelected(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbName</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>ExchangeEditDlg</receiver>
   <slot>OnNameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editKey</sender>
   <signal>editingFinished()</signal>
   <receiver>ExchangeEditDlg</receiver>
   <slot>OnKeyChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editSecret</sender>
   <signal>editingFinished()</signal>
   <receiver>ExchangeEditDlg</receiver>
   <slot>OnSecretChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDelete</sender>
   <signal>clicked()</signal>
   <receiver>ExchangeEditDlg</receiver>
   <slot>OnDelete()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNew</sender>
   <signal>clicked()</signal>
   <receiver>ExchangeEditDlg</receiver>
   <slot>OnNew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listExchanges</sender>
   <signal>itemEntered(QListWidgetItem*)</signal>
   <receiver>ExchangeEditDlg</receiver>
   <slot>OnExchangeSelected(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnClose()</slot>
  <slot>OnExchangeSelected(QListWidgetItem*)</slot>
  <slot>OnNameChanged(QString)</slot>
  <slot>OnKeyChanged()</slot>
  <slot>OnSecretChanged()</slot>
  <slot>OnDelete()</slot>
  <slot>OnNew()</slot>
 </slots>
</ui>
